examples/basic-webpack/index.jsx

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+var React = require('react');
+var ReactDOM = require('react-dom');
+
+var ExampleApplication = React.createClass({
+  render: function() {
+    var elapsed = Math.round(this.props.elapsed  / 100);
+    var seconds = elapsed / 10 + (elapsed % 10 ? '' : '.0' );
+    var message = 'React has been successfully running for ' + seconds + ' seconds.';
+
+    return (<p>{message}</p>);
+  }
+});
+
+var start = new Date().getTime();
+
+setInterval(function() {
```

Comment:
@keyanzhang Calling `ReactDOM.render` multiple times is not an anti-pattern.  In fact, if anything, calling setState would be the antipattern.  Ideally your components could all be stateless functions (ie. which have no `setState` function) and all your data would live in a flux store or something.